auth_oidc/controllers/main.py

```diff
@@ -48,3 +52,36 @@ def list_providers(self):
                     provider["auth_endpoint"], url_encode(params)
                 )
         return providers
+
+
+class OpenIDLogout(Session):
+    @http.route("/web/session/logout", type="http", auth="none")
+    def logout(self, redirect="/web/login"):
```

Comment:
Can you add a comment or docstring referencing the spec that this implements? 
https://openid.net/specs/openid-connect-rpinitiated-1_0.html#RPLogout